arrow-test/src/main/kotlin/arrow/test/laws/ShowLaws.kt

```diff
@@ -1,22 +1,23 @@
 package arrow.test.laws
 
 import arrow.typeclasses.Show
+import arrow.typeclasses.eq
 import arrow.typeclasses.show
 import io.kotlintest.properties.Gen
 import io.kotlintest.properties.forAll
 
 object ShowLaws {
 
     inline fun <reified F> laws(S: Show<F> = show<F>(), noinline cf: (Int) -> F): List<Law> =
-            listOf(
+            EqLaws.laws(eq<F>(), cf) + listOf(
```

Comment:
This eq has to come as a parameter to the laws, as we'd like for users to override it.